src/api/status/package.json

```diff
@@ -23,12 +21,19 @@
     "@octokit/core": "3.5.1",
     "@octokit/plugin-retry": "3.0.9",
     "@octokit/plugin-throttling": "3.5.2",
+    "@popperjs/core": "2.10.2",
     "@senecacdot/satellite": "^1.x",
+    "bootstrap": "5.1.3",
     "express": "4.17.2",
     "express-handlebars": "6.0.2",
     "npm-run-all": "4.1.5",
+    "perfect-scrollbar": "1.5.1",
     "sass": "1.45.2",
-    "vite": "2.7.13"
+    "vite": "2.7.13",
```

Comment:
If you're going to remove that, might as well move this to dev deps too.

Reply:
@manekenpix  should I change the docker file and install dev deps in prod as well?

Reply:
I love that you don't trust me :)

It's fine. The only thing it does is add extra space to the built image.  But I can remove that with a mult-layer build later on.

Reply:
How ironic, you are the only author of the file lol.